mdaviz/utils.py

```diff
@@ -81,22 +116,92 @@ def get_det(mda_file_data):
     P0.name = "Index"  # name of EPICS PV this positioner wrote to
```

Comment:
Why the text `"Index"` on both these lines?  (Does not look like a PV name here.)

Reply:
The rest of the code expects that every positioner/detector has a PV name. This is an equivalent to the "time" positioner (not a PV name but if you write "time" in the scan records field for positioner it actually works).